Core/Core/Api/GraphQL/Models/ViewerResourceGroup.cs

```diff
@@ -0,0 +1,11 @@
+﻿#nullable disable
```

Comment:
Best I explain this over a call, but the tldr is, because of the nature of GraphQL queries and responses makes nullability basically impossible have.